src/components/Product.js

```diff
@@ -1,8 +1,12 @@
 import React from "react";
 import Rating from "./Rating";
 import "../style/product.css";
+import {useCart} from '../context/cart-context'
+
+function Product({img, name, type, rating, price, data}) {
```

Comment:
Good that you have used destructing here

Reply:
Thank you